src/platforms/ros/nodes/commander/commander.cpp

```diff
@@ -205,8 +205,25 @@ void Commander::OffboardControlModeCallback(const px4::offboard_control_modeCons
 	/* Force system into offboard control mode */
 	if (!_got_manual_control) {
 		SetOffboardControl(_msg_offboard_control_mode, _msg_vehicle_control_mode);
+		
```

Comment:
@AndreasAntener This is the arming change I meant - that looks like it should work?